modules/snooker/src/main/ts/ephox/snooker/api/Generators.ts

```diff
@@ -77,18 +85,23 @@ const modification = (generators: Generators, toData = elementToData): Generator
     }
   };
 
-  const nuFrom = (element: SugarElement) => {
-    const data = toData(element);
-    return nu(data);
+  const nuRow = (data: RowData) => {
+    switch (SugarNode.name(data.element)) {
+      case 'colgroup':
+        return generators.colgroup(data);
+      default:
+        return generators.row(data);
+    }
   };
```

Comment:
For two options I think a ternary would be better here:
```
const nuRow = (data: RowData) => {
    const name = SugarNode.name(data.element);
    return name === 'colgroup' ? generators.colgroup(data) : generators.row(data);
};
```

Reply:
I honestly just copied this from `nuCell` above. I agree both would be better as a ternary.

Reply:
Fixed